lms/security.py

```diff
@@ -66,47 +67,60 @@ def get_route_policy(self, request):
         return self.fallback_policy()
 
 
-class LTISecurityPolicy:
+class _LTIUserBasedSecurityPolicy:
     @classmethod
-    def authenticated_userid(cls, request):
-        if request.lti_user is None:
-            return None
+    def get_lti_user(cls, request) -> LTIUser:
+        raise NotImplemented()
+
+    def permits(self, request, context, permission):
+        return _permits(self, request, context, permission)
+
+    def remember(self, request, userid, **kw):
```

Comment:
This is the default implementation (as opposed to declaring a method in an "abstract" class).